### ISSUE TITLE
az aks create --zones is confusing

### ISSUE THREAD

**Issue by mercer** [Azure/azure-cli#18101]
# TLDR

1. Please clarify `--zones` option with a better explanation
2. Please provide an example usage

# How does it work?

How is --zones supposed to work to get all 3 availability zones in a region with az?

1. is it `--zones 3`? (no, you only get zone 3)
2. is it `--zones 1,2,3`?  (no, does not work)
3. is it `--zones 123`?  (no, does not work)
4. is it `--zones 1 2 3`?  (no, does not work)
5. is it `--zones 1 --zones 2 --zones 3`?  (no, you only get zone 3)

For 1), 5) see yaml snippet bellow from generated resource:

```
        "agentPoolProfiles": [
            {
                "name": "nodepool1",
                "count": 1,
                "vmSize": "Standard_D4as_v4",
                "osDiskSizeGB": 50,
                "osDiskType": "Managed",
                "vnetSubnetID": "SOME-ID",
                "maxPods": 80,
                "type": "VirtualMachineScaleSets",
                "availabilityZones": [
                    "3"
                ],
                "provisioningState": "Succeeded",
                "powerState": {
                    "code": "Running"
                },
                "orchestratorVersion": "1.18.14",
                "enableNodePublicIP": false,
                "nodeLabels": {},
                "mode": "System",
                "osType": "Linux",
                "nodeImageVersion": "AKSUbuntu-1804gen2-2021.04.27"
            }
    ],
```

# Meanwhile, on terraform

Generated resource snippet:

```
    "agentPoolProfiles": [
      {
        "name": "nodepool",
        "count": 1,
        "vmSize": "Standard_D4as_v4",
        "osDiskSizeGB": 50,
        "osDiskType": "Managed",
        "vnetSubnetID": "SOME-ID",
        "maxPods": 80,
        "type": "VirtualMachineScaleSets",
        "availabilityZones": [
          "1",
          "2",
          "3"
        ],
        "enableAutoScaling": false,
        "provisioningState": "Succeeded",
        "powerState": {
          "code": "Running"
        },
        "orchestratorVersion": "1.18.14",
        "enableNodePublicIP": false,
        "nodeLabels": {},
        "mode": "System",
        "osType": "Linux",
        "nodeImageVersion": "AKSUbuntu-1804gen2-2021.04.27",
        "upgradeSettings": {}
      }
    ],
```

Snippet from terraform

```
  default_node_pool {
    name            = "nodepool"
    type            = "VirtualMachineScaleSets"
    node_count      = var.agents_count
    vm_size         = var.agents_size
    os_disk_size_gb = var.node_size
    vnet_subnet_id  = var.vnet_subnet_id
    max_pods        = var.max_pods_count
    availability_zones = ["1", "2", "3"]
  }
```

**Comment by yungezz** [comment]
route to appropriate team

**Comment by Volatus** [comment]
@mercer Pretty sure the usage is `--zones {1, 2, 3}`.

**Comment by swgriffith** [comment]
You can use:
ZONES=('1' '2')
 and then:
 az aks create -g $RG -n testcluster --zones $ZONES[@]
